imports/plugins/core/orders/client/containers/orderDashboardContainer.js

```diff
@@ -1,11 +1,13 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 import { Meteor } from "meteor/meteor";
+import { Tracker } from "meteor/tracker";
 import { composeWithTracker } from "@reactioncommerce/reaction-components";
-import { Media, Orders } from "/lib/collections";
+import { Media, Orders, OrderSearch as OrderSearchCollection } from "/lib/collections";
 import { Reaction, i18next } from "/client/api";
 import { Loading } from "/imports/plugins/core/ui/client/components";
```

Comment:
Is this still needed? Can we/should we use the `Components.Loading` version? I thought that the `Loading` component was included by default?